blinkpy/blinkpy.py

```diff
@@ -76,8 +76,10 @@ def start(self):
             self.get_auth_token()
 
         self.get_ids()
-        self.sync = BlinkSyncModule(self)
-        self.sync.start()
+        for network_id in self.network_ids:
+            sync_module = BlinkSyncModule(self)
```

Comment:
I think the network_id should be pass as parameter to BlinkSyncModule:
`sync_module = BlinkSyncModule(self, network_id)`

Reply:
Sorry, for me it is a bit complicated to setup a python environment :( I will try at least to analyze the code for the moment.